### ISSUE TITLE
SiGSEV on rss+xml Ubuntu feed

### ISSUE THREAD

**Issue by j-rivero** [guilhem/rss-issues-action#22]
Hello!

I'm trying to use the action in the repository (thanks for sharing) although I'm receiving a SIGSEV that I'm not able to know the root cause of it (limited Go knowledge sorry).

My testing setup is for checking nvidia package updates on Ubuntu Focal is as follow:
```
Run guilhem/rss-issues-action@0.5.1
  with:
    repo-token: ***
    feed: https://packages.ubuntu.com/focal/updates/newpkg?format=rss
    prefix: [Ubuntu Focal Nvidia 2022-05-02]
    characterLimit: 255
    dry-run: false
    lastTime: 24h
    labels: ubuntu
    titleFilteR: .*nvidia-graphics-drivers-390.*
/usr/bin/docker run --name ghcrioguilhemrssissuesaction051_f0b8b7 --label 294b31 --workdir /github/workspace --rm -e INPUT_REPO-TOKEN -e INPUT_FEED -e INPUT_PREFIX -e INPUT_CHARACTERLIMIT -e INPUT_DRY-RUN -e INPUT_LASTTIME -e INPUT_LABELS -e INPUT_TITLEFILTER -e INPUT_AGGREGATE -e INPUT_CONTENTFILTER -e HOME -e GITHUB_JOB -e GITHUB_REF -e GITHUB_SHA -e GITHUB_REPOSITORY -e GITHUB_REPOSITORY_OWNER -e GITHUB_RUN_ID -e GITHUB_RUN_NUMBER -e GITHUB_RETENTION_DAYS -e GITHUB_RUN_ATTEMPT -e GITHUB_ACTOR -e GITHUB_WORKFLOW -e GITHUB_HEAD_REF -e GITHUB_BASE_REF -e GITHUB_EVENT_NAME -e GITHUB_SERVER_URL -e GITHUB_API_URL -e GITHUB_GRAPHQL_URL -e GITHUB_REF_NAME -e GITHUB_REF_PROTECTED -e GITHUB_REF_TYPE -e GITHUB_WORKSPACE -e GITHUB_ACTION -e GITHUB_EVENT_PATH -e GITHUB_ACTION_REPOSITORY -e GITHUB_ACTION_REF -e GITHUB_PATH -e GITHUB_ENV -e GITHUB_STEP_SUMMARY -e RUNNER_OS -e RUNNER_ARCH -e RUNNER_NAME -e RUNNER_TOOL_CACHE -e RUNNER_TEMP -e RUNNER_WORKSPACE -e ACTIONS_RUNTIME_URL -e ACTIONS_RUNTIME_TOKEN -e ACTIONS_CACHE_URL -e GITHUB_ACTIONS=true -e CI=true -v "/var/run/docker.sock":"/var/run/docker.sock" -v "/home/runner/work/_temp/_github_home":"/github/home" -v "/home/runner/work/_temp/_github_workflow":"/github/workflow" -v "/home/runner/work/_temp/_runner_file_commands":"/github/file_commands" -v "/home/runner/work/security-advisories/security-advisories":"/github/workspace" ghcr.io/guilhem/rss-issues-action:0.5.1
New Ubuntu Packages
panic: runtime error: invalid memory address or nil pointer dereference
[signal SIGSEGV: segmentation violation code=0x1 addr=0x0 pc=0x7c36a7]

goroutine 1 [running]:
main.main.func1(0x1?)
	/src/main.go:102 +0x27
reflect.Value.call({0x7fb5a0?, 0xc000451c00?, 0xc0001859d8?}, {0x88320c, 0x4}, {0xc000185af0, 0x1, 0xc000185a08?})
	/usr/local/go/src/reflect/value.go:556 +0x845
reflect.Value.Call({0x7fb5a0?, 0xc000451c00?, 0x0?}, {0xc000185af0, 0x1, 0x1})
	/usr/local/go/src/reflect/value.go:339 +0xbf
github.com/thoas/go-funk.Filter({0x7eed40?, 0xc00048f9c8}, {0x7fb5a0?, 0xc000451c00})
	/go/pkg/mod/github.com/thoas/go-funk@v0.7.0/presence.go:41 +0x337
main.main()
	/src/main.go:101 +0x627
```

Am I doing some wrong with the configuration? Thanks!

**Comment by guilhem** [comment]
@j-rivero can you try to use action from #23 :
```
uses: docker://ghcr.io/guilhem/rss-issues-action:sha-238b655
```

**Comment by j-rivero** [comment]
> @j-rivero can you try to use action from #23 :

I did, and it works! I only have to fight now with the re2 regexp to exclude a single word, which seems not a trivial thing :)
Thank so much!

**Comment by guilhem** [comment]
Fixed in 0.5.2
